modules/ROOT/pages/configuration-schema-database.adoc

```diff
@@ -21,6 +21,9 @@ include::partial$block-caveats.adoc[tags="legacy-config-equivalents"]
 From {sgw-te} 3.0 you can use the Admin REST API to provision persistent configuration changes.
 This content introduces the <<lbl-create-db>> and <<lbl-configure-db>> endpoints for convenience -- see {rest-api-admin--xref--database-configuration} for a full description of the endpoints available.
 
+NOTE: You can define 1 custom scope per database with up to 1000 custom collections in release 3.1. +
```

Comment:
The `+` at the end of the line isn't needed I think?

Reply:
I'm not sure if "in release 3.1" is needed? 
Feels like the "oh yeah, we're only doing X, but Y will come later" is a bit specialist, and more for release notes or other reference?

but defer to your discussions with Iveta.